src/scales/scale.ts

```diff
@@ -22,11 +22,11 @@ module Plottable {
      * If one end of the domain is set to an excepted value as a result of autoDomain()-ing,
      * that end of the domain will not be padded.
      *
-     * @param {Scale} scale
-     * @returns {D[]} An array of extents.
+     * @param {QuantitativeScale} scale
+     * @returns {D[]} An array of values to become padding exceptions.
```

Comment:
They don't "become" padding exceptions, they *are* the padding exceptions.

I'd actually argue that the commentary isn't necessary, since the free-form comment block at the start of the documentation already explains what they are.